File: OneCodeTeam/LINQ-to-Objects-demoed-in-C-CSLinqToObject-16042/[C#]-LINQ to Objects demoed in C# (CSLinqToObject)/C#/CSLinqToObject/Person.cs
/****************************** Module Header ******************************\
* Module Name:  Person.cs
* Project:      CSLinqToObject
* Copyright (c) Microsoft Corporation.
* 
* This example illustrates how to write Linq to Object queries using Visual 
* C#. First, it builds a class named Person. Person inculdes the ID, Name and 
* Age properties. Then the example creates a list of Person which will be  
* used as the datasource. In the example, you will see the basic Linq 
* operations like select, update, orderby, max, average, etc.
* 
* This source is subject to the Microsoft Public License.
* See http://www.microsoft.com/en-us/openness/resources/licenses.aspx#MPL.
* All other rights reserved.
* 
* THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
* EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED 
* WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
\***************************************************************************/
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;


class Person
{
    public Person(int id, string name, int age)
    {
        this.id = id;
        this.name = name;
        this.age = age;
    }

    private int id;

    /// <summary>
    /// Person ID
    /// </summary>
    public int PersonID
    {
        get { return this.id; }
        set { this.id = value; }
    }

    private string name;

    /// <summary>
    /// Person name
    /// </summary>
    public string Name
    {
        get { return this.name; }
        set { this.name = value; }
    }

    private int age;

    /// <summary>
    /// Age that ranges from 1 to 100
    /// </summary>
    public int Age
    {
        get { return this.age; }
        set
        {
            if (value > 0 && value <= 100)
                this.age = value;
            else
                throw new Exception("Age is out of scope [1,100]");
        }
    }
}

File: OneCodeTeam/Call-web-through-service-bus-by-Windows-Phone-20226/[VB]-Call web services through service bus by Windows Phone Application/VB/ServiceBusServices/Person.vb
'****************************** Module Header ******************************\
'Module Name:  Person.vb
'Project:      ServiceBusServices
'Copyright (c) Microsoft Corporation.
'
'The sample code demonstrates how to expose an on-premise REST service to Internet
'via Service Bus, then you can access this service by Windows Phone application.
'The service includes normal string, generics and image methods.
'
'This is Person model class with some basic properties.
'
'This source is subject to the Microsoft Public License.
'See http://www.microsoft.com/en-us/openness/resources/licenses.aspx#MPL
'All other rights reserved.
'
'THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
'EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED 
'WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
'***************************************************************************/



Public Class Person
    Public Property Name() As String
        Get
            Return m_Name
        End Get
        Set(value As String)
            m_Name = value
        End Set
    End Property
    Private m_Name As String

    Public Property Comments() As String
        Get
            Return m_Comments
        End Get
        Set(value As String)
            m_Comments = value
        End Set
    End Property
    Private m_Comments As String
End Class
